CMakeLists.txt

```
@@ -117,6 +117,7 @@ option(VW_BOOST_MATH_SYS_DEP "Override using the submodule for boost math depend
 option(VW_ZLIB_SYS_DEP "Override using the submodule for zlib dependency. Instead will use find_package" ON)
 option(VW_GTEST_SYS_DEP "Override using fetch package for gtest dependency. Instead will use find_package" OFF)
 option(VW_BUILD_VW_C_WRAPPER "Enable building the c_wrapper project" ON)
+option(BUILD_CSV "Build csv parser" OFF)
```

maybe to match with the rest of the options:

```suggestion
option(VW_BUILD_CSV "Build csv parser" OFF)
```

---

Fixed!